src/main/java/org/broadinstitute/hellbender/utils/codecs/gtf/AbstractGtfCodec.java

```diff
@@ -521,6 +515,25 @@ boolean isLineCommented(final String line) {
         return isCommented;
     }
 
+    /**
+     * Checks whether the given line is commented out or not and if the content of the line begins with the given
```

Comment:
shouldn't the other other implementation make use of this one now?

Reply:
Oops.  Yes it should!

Reply:
Fixed.